### DISCUSSION TITLE
RPZ Hits

### DISCUSSION THREAD

**Discussion by enkrispi** [dmachard/DNS-collector#798] [Q&A]
Hi all, is there an option to show rpz hits in Prometheus collector ? I'm currently collecting powerdns recursor using protobuf but when i try to lookup the metrics it seems rpz is not showing up, it works when in stdout and adding text-format `policy-match`.
Do I need to change the logger ?

Thanks in advance

**Comment by dmachard** [comment]
Hi, the Prometheus exporter doesn't support metrics for PowerDNS.
Could you open a feature request?

**Reply by enkrispi** [reply]
Done #807 
Thank you in advance
